code/datums/mood_events/drug_events.dm

```diff
@@ -65,12 +65,12 @@
 	timeout = 3 MINUTES
 
 /datum/mood_event/narcotic_heavy
-	description = "<span class='nicegreen'>I feel like I'm wrapped in cotton!</span>\n"
+	description = "<span class='nicegreen'>I feel like I'm wrapped up in cotton!</span>\n"
 	mood_change = 9
 	timeout = 3 MINUTES
 
 /datum/mood_event/stimulant_medium
-	description = "<span class='nicegreen'>I have so much energy and I feel like I could do anything.</span>\n"
+	description = "<span class='nicegreen'>I have so much energy! I feel like I could do anything!</span>\n"

```

Comment:
This one is worth changing, the other ones are code churn. Ln 103 is actually being made inaccurate, as it is referring to an event that just occurred rather than the head itself.

Reply:
I understand, but I felt for the other lines of code I had modified, before when I read it. It didn't sound right in my view.

Reply:
I get that, I suppose it comes down to style choices.

Reply:
> 
> 
> I get that, I suppose it comes down to style choices.

Yeah